### DISCUSSION TITLE
How do I run the RBibTeXK command?

### DISCUSSION THREAD

**Discussion by lokesh-krishna** [R-nvim/R.nvim#198] [Q&A]
I have a Quarto document that I'd like to render and preview with synctex support. The `RBibTeXK` command is not bound to any keymapping by default. I attempted to add one with `vim.keymap.set('n', '<LocalLeader>rk', '<Plug>RBibTeXK<cr>', { desc = 'Generate and preview pdf' })` but it doesn't do anything. Am I missing something?

**Comment by lokesh-krishna** [comment]
Sorry, missed the actual command required and that it is setup. My bad.

**Comment by jalvesaq** [comment]
The output of

```
grep -R RBibTeXK
```

is:

```
lua/r/maps.lua:    RBibTeXK            = { m = "", k = "", c = "Weave",    d = "Knit the document, run bibtex and generate the PDF" },
lua/r/maps.lua:        create_maps("nvi", "RBibTeXK",     "kb", "<Cmd>lua require('r.rnw').weave('bibtex', true, true)")
```

As with Nvim-R, `<LocalLeader>kb` works in `Rnoweb` documents, not Quarto ones, and there is no SyncTeX support for Quarto because we would have to find in the Quarto document the corresponding line from the LaTeX intermediate document. It's not impossible to implement something like this, but it would be better if this were supported by `quarto preview`, independent of the editor being used.

**Reply by lokesh-krishna** [reply]
Thanks a lot for clarifying this :D
